components/lcd/esp_lcd_ssd1681/idf_component.yml

```
@@ -0,0 +1,5 @@
+version: "0.1.0"
+description: ESP LCD SSD1681
```

let's high light it's an e-paper driver
```suggestion
description: ESP LCD SSD1681 e-paper driver
```